src/shared/components/search-result/search-result.component.scss

```diff
@@ -6,6 +6,7 @@
 $blue_table: #005ea5;
 $blue_pagination: #005da6;
 $black: #231F20;
+$focus-yellow: #FFDD00;
```

Comment:
Why not use `$govuk-focus-colour` as it's already part of the GDS colour palette?

Reply:
This is what I talked about in the tech forum. The colour that comes through on govuk-focus-colour is an 'orangey' yellow and does not match the rest of the website if I do so.

Reply:
Oh, according to the current GDS style guide, that's the same hex colour. We clearly have an out-of-date one somewhere!